lib/LaTeXML/Package/AmSTeX.pool.ltxml

```diff
@@ -193,6 +194,14 @@ Let('\D',           "\\.");
 
 DefMacro('\.', '. ');
 
+# instead of adding taylor's diagram.tex
+# for now warn that we are aware of those diagrams, but they are not supported
+# and avoid the sea of otherwise inevitable errors during drawing
+DefMacro('\diagram Until:\enddiagram', sub {
+    Warn('missing', 'support', 'The \diagram mechanism of diagram.tex is not currently supported, output is degraded.');
+    return Tokenize("missing diagram")->unlist(); },
+  locked => 1);
```

Comment:
btw I did end up adding `diagram.tex.ltxml` in the arxmliv-bindings repo, where I defined:
```
# see arXiv:math/0002180 for an ambitious use.
DefEnvironment('{diagram}', 'diagram.tex is currently unsupported', locked=>1);
```

Wondering if I should delete the guard in AmSTeX.pool.ltxml, though nothing really wrong with guarding in two separate ways (maybe a different package has an undefined \diagram?)